src/qml/FeatureForm.qml

```diff
@@ -253,6 +253,8 @@ Page {
 
         Loader {
           id: attributeEditorLoader
+          asynchronous: true
+          visible: status == Loader.Ready
```

Comment:
that's one thing which helped.
As far as I tested it it doesn't cause any issue.

Reply:
woohoo